src/client/shared/ui/ui-svg-icon/styles/svg-icon-sizes.scss

```
@@ -0,0 +1,20 @@
+  'lg':      (2.5rem,  1.5rem, 0),
+);
+
+@mixin icon-size($height, $padding-x, $padding-y: 0) {
+  height: $height;
+  padding: $padding-y $padding-x;
+}
+
+.icon-svg {
+  @each $name, $values in $sizes {
+    &[data-size='#{$name}'] {
+      @include icon-size(nth($values, 1), nth($values, 2), nth($values, 3));
```

Stylelint (17.5.0)

[error] Expected list.nth instead of nth (scss/no-global-function-names)

(scss/no-global-function-names)